### PR TITLE
Improved ci

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Diverses CI improvement, cf commit list.

If you merge this, you would have to activate coveralls: https://coveralls.io/github/jamestalmage/istanbul-combine?branch=master

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -2,7 +2,10 @@ istanbul-combine
 ================
 Combine coverage reports from multiple runs in to a single coverage report.
 
+[![Npm Version](https://img.shields.io/npm/v/istanbul-combine.svg)](https://npmjs.com/package/istanbul-combine)
```
Comment (jamestalmage) [inline Readme.md:5]: No NPM badge, please. I prefer to keep it to just the important ones.

File: Readme.md [hunk 2]
```diff
@@ -2,7 +2,10 @@ istanbul-combine
...
 [![Build Status](https://travis-ci.org/jamestalmage/istanbul-combine.svg?branch=master)](https://travis-ci.org/jamestalmage/istanbul-combine)
+[![Coverage Status](https://coveralls.io/repos/jamestalmage/istanbul-combine/badge.svg?branch=master&service=github)](https://coveralls.io/github/jamestalmage/istanbul-combine?branch=master)
```
Comment (jamestalmage) [inline Readme.md:7]: Coverage is meaningless on this project. This would only report the coverage for the fixtures.

File: package.json [hunk 3]
```diff
@@ -26,12 +27,13 @@
...
   "dependencies": {
-    "glob": "^5.0.3",
+    "glob": "^6.0.4",
     "minimist": "^1.1.1",
     "q": "^1.2.0",
-    "istanbul": "0.3.x"
+    "istanbul": "0.4.x"
   },
   "devDependencies": {
+    "coveralls": "^2.11.6",
```
Comment (jamestalmage) [inline package.json:36]: No coverage.

File: package.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "rm -rf coverage && npm run test-a && npm run test-b && node ./fixture/combine.js",
+    "coverage": "istanbul cover _mocha && cat ./coverage/lcov.info | coveralls",
```
Comment (jamestalmage) [inline package.json:8]: no coverage / coveralls

File: Readme.md (hunk 1 above)
Comment (AdrieanKhisbe) [inline Readme.md:5]: npm version badge is to me one of the most important one.

**Comment by AdrieanKhisbe** [comment]
Applying your request the PR would be meaningless. 
Hence closing

**Pull request closed without merging**